loadtest/control/utils.go

```diff
@@ -81,3 +82,29 @@ func emulateUserTyping(t string, cb func(term string) UserActionResponse) UserAc
 	}
 	return resp
 }
+
+// GenerateRandomSentences generates random string from test_text file.
+func GenerateRandomSentences(count int) string {
+	if count <= 0 {
+		return "🙂" // if there is nothing to say, an emoji worths for thousands
+	}
+	paths := []string{"./testdata/test_text.txt", "./../../testdata/test_text.txt"}
+	var buf []byte
+	var err error
+
+	for _, p := range paths {
+		buf, err = ioutil.ReadFile(p)
```

Comment:
We can also move this to init function maybe, with that we have the ability to test with different languages as well. Thoughts?

Reply:
That can work :+1:

Reply:
I would suggest avoiding the file because that's yet another dependency to run the test and it also means we cannot run the test without the codebase. A const string is better.

Ideally, this should be part of an asset generation step using "go generate", but we can come to it later. For now, I think a const string is fine.

Reply:
Since we added 10K most used words, I think it is better to stay as a separate file. With this, we may have the possibility to change language, words etc.

Reply:
Ok, but then please also create a ticket to make this into an asset generation step. Because right now, you cannot run the test without the codebase. This is going to be problematic when we start deploying load test agents.

I have to even asset-ize the terraform file.

Reply:
Will do 👍